package.json

```diff
@@ -85,7 +85,7 @@
     "url": "https://github.com/brave/brave-core-crx-packager.git"
   },
   "resolutions": {
-    "ansi-regex": "6.0.1"
+    "ansi-regex": "5.0.1"
```

Comment:
is this downgrade intentional?

Reply:
Yes, that was the cause of the build failures in https://github.com/brave/brave-core-crx-packager/pull/412.

From what I could tell, forcing this new version (done by Renovate in https://github.com/brave/brave-core-crx-packager/pull/325) is causing incompatibilities. It's also not required to fix any `npm audit` failures anymore.